packages/lesswrong/components/common/TabNavigationMenu/TabNavigationMenu.tsx

```diff
@@ -17,6 +18,12 @@ const styles = (theme: ThemeType): JssStyles => {
       flexDirection: "column",
       justifyContent: "space-around",
       maxWidth: TAB_NAVIGATION_MENU_WIDTH,
+      paddingTop: 15,
+    },
+    navSidebarTransparent: {
+      zIndex: 10,
+      background: `${theme.palette.background.default}cf`, // Add alpha to background color, not thrilled about this way of doing it
```

Comment:
Eh, it's probably fine (well, at least until someone changes it to not be a hex colour, but I thik that's pretty unlikely).

If you think it's an issue we could just add a new colour to the pallette and define it next to this one to make it obvious they need to be updated together.

Reply:
Yeah I thought about doing that, we already have one called `translucentBackground` though and I thought adding `translucentBackground2` would be no less confusing than doing it this way